platforms/hyperledger-fabric/configuration/roles/create/cli_pod/tasks/main.yaml

```diff
@@ -0,0 +1,45 @@
+#############################################################################################
+# This role creates value file for Cli pods
+############################################################################################
+#                      CREATE CLI POD                                                      #
+############################################################################################
+#
+############################################################################################
+# This task creates the value file for the cli pod as per requirements mentioned in 
+# network.yaml
+- name: "Create Value file for CLI Pod"
+  include_role:
+    name: helm_component
+  vars:
+    name: "cli"
+    component_name: "{{ org.name | lower}}-cli"
+    orderer_image: "hyperledger/fabric-orderer:{{ network.version }}"
+    alpine_image: "{{ docker_url }}/alpine-utils:1.0"  
+    peer: "{{ org.services.peers | first }}"
```

Comment:
This is wrong. 
CLI is per peer, not per organization

Reply:
So do you want cli for every peer  in a particular org?